workbench-core/environments/src/postDeployment/cognitoSetup.test.ts

```diff
@@ -15,8 +16,36 @@ describe('CognitoSetup', () => {
   const constants = {
     AWS_REGION: 'us-east-1',
     ROOT_USER_EMAIL: 'user@example.com',
-    USER_POOL_NAME: 'swb-userpool-test-va'
+    USER_POOL_NAME: 'swb-userpool-test-va',
+    STACK_NAME: 'swb-dev-test'
   };
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function mockCloudformationOutputs(cfMock: AwsStub<any, any>): void {
+    cfMock.on(DescribeStacksCommand).resolves({
+      Stacks: [
+        {
+          StackName: 'swb-dev-va',
```

Comment:
since you've set up constant.STACK_NAME, you can call it here to be a consistent mock value